web/src/lazy/zxcvbn.js

```diff
@@ -0,0 +1,9 @@
+let zxcvbn = () => ({ score: 0 });
```

Comment:
This file does the lazy loading.  It starts by creating a temporary object that does some default behavior for the things we need (we only use `zxcvbn().score`, so that's the only thing provided here).  Then once the real one is loaded, it replaces the temporary object with the real one.